src/frontend/src/instance/log_handler.rs

```diff
@@ -103,6 +104,18 @@ impl PipelineHandler for Instance {
     fn build_pipeline(&self, pipeline: &str) -> ServerResult<Pipeline> {
         PipelineOperator::build_pipeline(pipeline).context(PipelineSnafu)
     }
+
+    async fn get_original_pipeline(
```

Comment:
Nitpick: I would prefer `get_pipeline_str`, what do you think @paomian @zyy17

Reply:
Hmm, I will updated this name in the latter commit soon.